sound/soc/sof/sof-audio.h

```diff
@@ -97,8 +97,6 @@ struct ipc_tplg_control_ops {
  * @token_list: List of token ID's that should be parsed for the widget
  * @token_list_size: number of elements in token_list
  * @bind_event: Function pointer for binding events to the widget
- * @set_up_all_pipelines: Function pointer for setting up all topology pipelines
- * @tear_down_all_pipelines: Function pointer for tearing down all topology pipelines
  */
```

Comment:
It looks like fixup for https://github.com/thesofproject/linux/commit/06bf35b2a41ffa346f0b2521b0378101594d11bd and https://github.com/thesofproject/linux/commit/7c81c82d4e667e90ed064ac712e57337f9379b97. Should we separate it into two commits? So we will squash them before sending upstream.